compose.go

```diff
@@ -76,6 +76,56 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	composeCmd.Flags().BoolVar(&conflictsVerbosity, "conflicts-verbosity", false, "Log files conflicts")
 	composeCmd.Flags().BoolVar(&clean, "clean", false, "Remove .compose dir on start")
 	composeCmd.Flags().BoolVar(&interactive, "interactive", true, "Interactive mode allows to submit user credentials during action")
+
+	composeDependency := &compose.Dependency{}
+	var createNew bool
+	var addCmd = &cobra.Command{
+		Use:   "compose:add",
+		Short: "Add new entry to plasma-compose",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.AddPackage(createNew, composeDependency, p.wd)
```

Comment:
On this step : 
```
  - Enter Tag 
  > 1.23.0

  - Enter Ref
  >
```
It's possible for user to define both a `tag` or a `ref`, whether we should have either one or the other

So I suggest to have a selector in first "step" of package addition:
```
  - Enter package name
  >

  - Select source type
  > Git
    Http

  - Enter package URL
  >

  - Select source reference
  > Tag
    Ref
```